src/moby/config.go

```diff
@@ -705,6 +705,8 @@ func ConfigInspectToOCI(yaml *Image, inspect types.ImageInspect, idMap map[strin
 	var inspectConfig container.Config
```

Comment:
I think its cleaner if you just replace this with `inspectConfig := &container.Config{}` here.

Reply:
makes sense.

Reply:
* and done :)